src/main/cliapp/src/containers/dataLoadsPage/NewBulkLoadForm.js

```diff
@@ -170,6 +170,18 @@ export const NewBulkLoadForm = ({ bulkLoadDialog, setBulkLoadDialog, groups, new
                         />
                     </div>
 
+					<div className="p-field">
+						<label htmlFor="fileExtension">File Extension</label>
+						<InputText
+							id="bulkLoadFileExtension"
+							value={newBulkLoad.fileExtension}
+							onChange={onChange}
+							placeholder={"Select Bulk Load File Extension"}
+							name='fileExtension'
+							disabled={disableFormFields}
```

Comment:
@markquintontulloch sorry last item... not sure if this field is specified somewhere else? Maybe disableFormFields is just a boolean.

Reply:
This is used so that fileExtension can't be changed after files have been uploaded.

Reply:
isn't that what we want, otherwise the filename won't match the saved file?

Reply:
Correct... I assume that disableFormFields is a boolean and not a list of fields to disable. If it is a list, then this field would need to be specified in another place to "disable" after editing @adamgibs ? Maybe it's just late and I need to look at this in the morning.

Reply:
It does disable the field from being edited on my local system, but I'll let @adamgibs take a look before merging in case I'm doing something funky!

Reply:
Oh ok... then yes looks good